.github/ISSUE_TEMPLATE/failing-test.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 🚨 Failing Test
```

Comment:
Could we add this to CAPZ? Very helpful

Reply:
Sure thing, I just copied this from CAPI. I'm not sure if we really need it in this repo ATM, but it would definitely be useful for CAPZ.